packages/core-ethereum/src/indexer/reducers.ts

```diff
@@ -1,153 +1,174 @@
-import type { Event } from './topics'
+import type { Event } from './types'
 import assert from 'assert'
 import BN from 'bn.js'
-import { ChannelEntry } from '../types'
+import { publicKeyConvert } from 'secp256k1'
+import { stringToU8a } from '@hoprnet/hopr-utils'
+import { AccountEntry, Address, Public, Hash, ChannelEntry } from '../types'
 import { isPartyA } from '../utils'
 
-export const onFundedChannel = async (
-  event: Event<'FundedChannel'>,
+export const onAccountInitialized = async (event: Event<'AccountInitialized'>): Promise<AccountEntry> => {
+  const data = event.returnValues
+  const address = Address.fromString(data.account)
+  // library requires identifier TODO: investigate why
+  const pubKey = new Public(publicKeyConvert(stringToU8a('0x04' + data.uncompressedPubKey.slice(2)), true))
```

Comment:
This is a prefix byte used to identify the public key, see https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L179-L184 for examples of other values cc @robertkiel

Reply:
yeah I know, I guess I expected the library to not require an identifier, anyway I need to update this so it simply inserts the byte and not doing this string workaround